### DISCUSSION TITLE
Classification of data

### DISCUSSION THREAD

**Discussion by Camulos** [digdir/designsystemet#1802] [Patterns for public services]
At Brønnøysund Register Centre we have a lot of data that is made accessible in various ways. some of those data is open for everyone to access and use as they want. Some data is available if certain conditions is met (like you have to login or confirm your role), and a third group of information is only available to whoever the data is about (and a few others). Internally we call these groups of data green, yellow and red data, but externally for the general public these words might not be as easy to understand.

We suggest that we find a similar way or pattern to describe these types of data so that people (the generall public) can understand.

**Comment by Febakke** [comment]
Is this something that would be relevant for: https://data.norge.no/ ?

**Reply by Camulos** [reply]
Very likely

**Reply by mrosvik** [reply]
@KjerstiSteien

**Reply by KjerstiSteien** [reply]
@Camulos I am so sorry that I did not see this mention until now. Data.norge.no already has a pattern that it should be able to map from your green, yellow and red groups of data. 
Green: "Allmenn tilgang"/"Public" (This access right in combination with an open licence, gets an additional tag, "Åpne data"/"Open data"
Yellow: "Betinget tilgang"/"Restricted"
Red: "Ikke-allmenn tilgang"/"Non-public"

You can read more about the traffic light model here: https://www.digdir.no/datadeling/sjekkliste-datakonsument/2269#steg_3_be_om_tilgang_til_data
Here is a descriotion of how this information should be expressed in dcat-ap-no: https://data.norge.no/specification/dcat-ap-no#Datasett-tilgangsrettigheter
And this is the codelist from EU: https://op.europa.eu/en/web/eu-vocabularies/concept-scheme/-/resource?uri=http://publications.europa.eu/resource/authority/access-right
